internal/policy/verify.go

```diff
@@ -169,7 +169,7 @@ func VerifyRelativeForRef(ctx context.Context, repo *git.Repository, initialPoli
 			slog.Debug(fmt.Sprintf("Verifying entry '%s'...", entry.ID.String()))
 
 			slog.Debug("Checking if entry is for policy reference...")
-			if entry.RefName == PolicyRef {
+			if entry.RefName == PolicyRef || entry.RefName == PolicyStagingRef {
```

Comment:
No, we should never be loading the policy staging ref during verification.

Reply:
staging should be special cased so we skip to the next one.

Reply:
Fixed